contributor-guide/modules/ROOT/pages/contributors-faq.adoc

```diff
@@ -0,0 +1,129 @@
+* boost:compute[] provides abstractions for memory management, kernel execution, and data parallelism, enabling developers to harness the computational power of modern hardware for tasks such as numerical simulations, image processing, and machine learning.
+* boost:spirit[] is ambitious in its goal of providing a high-level and composable framework for parsing complex data formats and domain-specific languages entirely within pass:[C++] code, without the need for external tools or preprocessors.
+* boost:hana[] aims to simplify and modernize metaprogramming in pass:[C++], making it more accessible and powerful for developing generic libraries and applications.
+
+. *What libraries were the least ambitious technically?*
++
+The useful utilities such as boost:any[], boost:variant[], and boost:optional[] offer relatively simple functionality. Another simpler library is boost:bimap[] which provides a container for maintaining one-to-one mappings between keys and values. While bidirectional maps are a useful data structure, the functionality provided is relatively straightforward and focused on this specific use case.
+
+== Modular Boost
+
+. *What is meant by "Modular Boost"?*
++
+Technically, Modular Boost consists of the Boost super-project and separate projects for each individual library in Boost. In terms of Git, the Boost super-project treats the individual libraries as submodules. Currently (early 2024) when the Boost libraries are downloaded and installed, the build organization does _not_ match the modular arrangement of the Git super-project. This is largely a legacy issue, and there are advantages to the build layout matching the super-project layout. This concept, and the effort behind it, is known as "Modular Boost".
++
+Refer to xref:superproject/overview.adoc[] for a full description of the super-project.
+
+. *What exactly is a "modular arrangement"?*
++
+It's when the libraries can be used, and hence built, without creating the monolithic headers, without needing the root build files, and without needing the libraries to be arranged in the usual `root/libs/<name>` format.
+
+. *Will the move to Modular Boost change testing?*
++
+No, unless you want to. You will still be able to test with the current non-modular way. But you could also test the modular way.
+
+. *How will modular Boost work if there is no `root/libs/<name>` structure? Or is the structure still required?*
++
+The structure is still required for things like testing and documentation building.
+
+. *What happens to the numeric libraries that are currently sub-libraries, when sub-libraries are no longer supported?*
++
+The numeric libraries have been divided into four packages: libboost-numeric-conversion/, libboost-numeric-interval/, libboost-numeric-odeint/, libboost-numeric-ublas/.
+
+== New Libraries
```

Comment:
These questions should probably go first?

Reply:
The sections are alphabetized, however by changing the title to "Adding New Libraries" this section comes first.

Reply:
> The sections are alphabetized

Oh... I see. I think FAQs should attempt to list the most frequently questions first.